webrender/src/render_backend.rs

```diff
@@ -261,7 +265,16 @@ impl RenderBackend {
                         }
                         ApiMsg::RequestWebGLContext(size, attributes, tx) => {
                             if let Some(ref wrapper) = self.webrender_context_handle {
-                                let result = wrapper.new_context(size, attributes);
+                                let dispatcher: Option<Box<GLContextDispatcher>>;
+                                if cfg!(target_os = "windows") {
+                                    dispatcher = Some(Box::new(WebRenderGLDispatcher {
```

Comment:
I need to specify the type: Option<Box<GLContextDispatcher>> or rust compiler doesn't have the enough magic to coerce